### DISCUSSION TITLE
Mise à jour 1.3

### DISCUSSION THREAD

**Discussion by BlastCom** [Ludeon/RimWorld-fr#337] [General]
Bonjour tout le monde!

Je m'appelle BlastCom, et je vous écris aujourd'hui pour discuter de la traduction de la mise à jour 1.3 de Rimworld qui a été annoncée hier et de ma contribution futur.

J'ai un peu participer aux traductions durant l'hiver 2019, mais sans plus.

Depuis 2 mois, je traduis des mods et j'ai gagné de l'expérience non seulement en traduction, mais comment fonctionne la hiérarchie des fichiers de traduction.

J'aimerais participer davantage à la prochaine traduction de la mise à jour qui se prépare dans un avenir proche. (La bêta semblerait déjà être disponible)

Le seul bémol qui m'empêche de participer davantage est la communication dans ce groupe. Est-ce qu'il y a un discord ( ou tout simplement ici dans le canal de discussion ) pour attribuer des tâches ou c'est premier arrivé premier servi?
De plus, je peux seulement publier en Pull-Request qui ralentit un peu le procédé, mais je comprends tout à fait la raison de celà.

Merci et bonne journée à toutes!

**Comment by b606** [comment]
Bonjour BlastCom,

Je vais essayer de voir çà demain mais je risque d'être peu réactif dans les quelques jours qui suivent. Il faudrait voir avec Tynan si tu peux faire directement un push sur le master.

Il y a bien un discord Rimworld-fr que **Boundir** a ouvert. Un lien d'invitation doit se trouver quelque part sur https://ludeon.com/forums/index.php?topic=2977.315  mais pour l'instant je n'ai pas remis la main dessus (Bug: On aurait dû l'écrire en grand sur la page home ici).
A+

**Reply by BlastCom** [reply]
Merci !
J'ai envoyé un courriel à Ludeon et je suis présentement contributeur :)
J'ai créé une branche séparée nommée "RW1.3"
Je voulais pas modifier la branche "unstable" pour pas perdre les données de la version 1.2

À propos du Discord, j'ai cherché et rien trouvé. Pour l'instant je vais démarrer des disscusions pour la traduction dans l'onglet Disscusions

**Reply by boundir** [reply]
Salut BlastCom, b606,

Bienvenue dans l'équipe! On a effectivement un discord : discord.gg/cJmKZpv

**Comment by aleshaglenna26** [comment]
ok

**Comment by AbbeFriend29** [comment]
slay
